<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POIPlugin</class>
 <widget class="QWidget" name="POIPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POIPlugin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ID">
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string>测点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_XValue">
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string>X位移(mm)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_YValue">
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string>Y位移(mm)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ZValue">
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string>Z位移(mm)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/BiVMS2020.qrc">
         <normaloff>:/test/BiVMS/delete (1).png</normaloff>:/test/BiVMS/delete (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/BiVMS2020.qrc"/>
 </resources>
 <connections/>
</ui>
